app/views/partials/_home_icons.html.erb

```diff
@@ -0,0 +1,67 @@
+<div class="container mt-5">
+  <div style="text-align: center">
+  <h3>You bring the brains, we’ll take care of everything else.
+</h3>
+  <h1> The greatest learning experience, join us to discover more.</h1>
+</div>
+  <div class="row mt-5">
+    <div class="col-sm-6">
+      <div style="padding: 2em">
+        <h4>You bring the brains, we’ll take care of everything else</h4>
+        <p> From flashcards to help you learn French to games that make it easy to understand history, you can use a variety of tools to conquer any challenge. </p>
+    </div>
+    </div>
+
+    <div class="col-sm-6">
+        <div style="text-align: center; padding: 2em">
+          <i class="far fa-address-card" style="font-size: 1200%" important!></i>
```

Comment:
!important  should be inside the "" of style
style = "font-size: 1200% !important;"

Reply:
implemented that, code looks better now